lib/qa/authorities/linked_data/config/search_config.rb

```diff
@@ -89,31 +73,26 @@ def results_sort_predicate
         Config.predicate_uri(results, :sort_predicate)
       end
 
-      # Return parameters that are required for QA api
-      # @return [Hash] the configured search url parameter mappings
-      def qa_replacement_patterns
-        search_config.fetch(:qa_replacement_patterns)
-      end
-
-      # Are there replacement parameters configured for search query?
-      # @return [True|False] true if there are replacement parameters configured for search query; otherwise, false
-      def replacements?
-        replacement_count.positive?
+      # Does this authority configuration support additional context in search results?
+      # @return [True|False] true if additional context in search results is supported; otherwise, false
+      def supports_context?
+        return true if context_map.present?
+        false
       end
 
-      # Return the number of possible replacement values to make in the search URL
-      # @return [Integer] the configured number of possible replacements in the search url
-      def replacement_count
-        replacements.size
+      # Return the context map if it is defined
+      # @return [Qa::LinkedData::Config::ContextMap] the context map
+      def context_map
+        return @context_map if @context_map.present?
+        context_config = search_config.fetch(:context, {})
+        return nil if context_config.blank?
+        @context_map = Qa::LinkedData::Config::ContextMap.new(context_config)
       end
 
-      # Return the replacement configurations
-      # @return [Hash] the configurations for search url replacements
-      def replacements
-        return @replacements unless @replacements.nil?
-        @replacements = {}
-        @replacements = url_mappings.select { |k, _v| !qa_replacement_patterns.include?(k) } unless search_config.nil? || url_mappings.nil?
-        @replacements
+      # Return parameters that are required for QA api
+      # @return [Hash] the configured search url parameter mappings
+      def qa_replacement_patterns
+        search_config.fetch(:qa_replacement_patterns)
```

Comment:
#replacements?, #replacement_count, #replacements are no longer used.  The processing of replacements happens in AuthorityUrlService and IriTemplate::UrlConfig, and IriTemplate::VariableMap.  These methods and there tests have been removed.